contrib/sarplus/scala/python/setup.py

```diff
@@ -1,8 +1,13 @@
 from distutils.core import setup
+import os
+
+version = os.getenv("VERSION")
+if version is None:
+    version = "0.5.0"
```

Comment:
should we remove the default and force version to exist the same way we doing in the python/setup.py?

Reply:
The code is used [here when building the package](https://github.com/simonzhaoms/recommenders/blob/simonz/sarplus/spark3/contrib/sarplus/DEVELOPMENT.md#notes-for-spark-3x)

Reply:
Any suggestions on how to or where to specify the version?

Reply:
keeping the version as an environment variable is fine. I just meant that line 10 pulls the version from the environment as well, so you don't need these lines (4-6)

Reply:
Removed

Reply:
Changed